index.html

```diff
@@ -896,7 +898,14 @@
           [SameObject] readonly attribute PushSubscriptionOptions options;
           ArrayBuffer? getKey(PushEncryptionKeyName name);
           Promise&lt;boolean&gt; unsubscribe();
-          serializer;
+
+          PushSubscriptionJSON toJSON();
+        };
+
+        dictionary PushSubscriptionJSON {
+          USVString endpoint;
+          DOMTimeStamp? expirationTime;
+          record keys;
```

Comment:
So this should probably be `record<USVString, USVString>` (or `DOMString`?—I must admit still being unsure when to use which.)